Makefile

```diff
@@ -35,8 +35,8 @@ wasm-test:
 gosdk-mocks:
 	./generate_mocks.sh
 
-gosdk-test: wasm-test
-	go test -v -tags bn256 $(shell go list ./... | grep -v wasm)
+gosdk-test:
```

Comment:
did we remove the wasm test in ci?

Reply:
gosdk-test and wasm-test can't run on same environment. so there is a new ci job named `wasm-test` on github action.